apps/server/src/app/import.controller.ts

```diff
@@ -158,7 +158,11 @@ export class ImportController extends BaseController {
     @User() user: ITofUser,
     @Headers('vsacauthorization') vsacAuthorization: string,
     @Param('id') id: string,
-    @Param('applyContextPermissions') applyContextPermissions = true) {
+    @Param('applyContextPermissions') applyContextPermissions = true,
+    @Query('implementationGuideId') contextImplementationGuideId) {
```

Comment:
I think you can get the context implementation guide from `@RequestHeader('implementationGuideId')` already, and don't need to add a new parameter for it... The interceptor _should_ automatically add the context implementation guide to the headers on _every_ request to `/api/**`.

Reply:
Tried the change and it worked as you expected.